bower.json

```diff
@@ -83,8 +86,11 @@
     "knockout": {
       "js": "dist/knockout.debug.js"
     },
-    "lru-ts": {
-      "js": "dist/global/lru-ts.js"
+    "lodash": {
+      "js/": "dist/*.js"
```

Comment:
underscore and losh using the same `_` object so we should either ensure to use noConflict version or remove on of them